### DISCUSSION TITLE
java.lang.RuntimeException: Unable to get provider

### DISCUSSION THREAD

**Discussion by dan19-82** [pichillilorenzo/flutter_inappwebview#2427] [Q&A] [answered]
I have spent ages trying to work out why this error was occurring and crashing my app on load. Might seem obvious to someone that the provider name was wrong, however I copied all the installation instructions from the website so they shouldn't have been wrong. However i eventually worked out that the actually name of the provider is android:name="com.pichillilorenzo.flutter_inappwebview_android.InAppWebViewFileProvider" (note the _android)

Is there any reason why this is different for me, did i do something to cause this?

**Comment by pichillilorenzo** [comment, marked as answer]
If you are upgrading from version 5, have you checked the migration guide from version 5?
The package name change is one of the first things written there: https://inappwebview.dev/docs/migration-guide
Also, the Android config docs of version 6 is already correct: https://inappwebview.dev/docs/intro#configure-android

**Reply by dan19-82** [reply]
I guess i had not. I had just done a pub upgrade and this must have caused the problem. I had no idea that there was such a thing. I appreciate this was my fault but at least there is a reason. I maybe hope that this helps someone else if they do a google search or something, cause i've been stuck for about a day.
